ISSUE_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 
 # Proposal: <proposal name>
 
-*Author*: Your Name <@yourGHid> IRC: handle (if different)
+*Author*: Your Name <@yourGithubID> IRC: handle (if different)
```

Comment:
```suggestion
*Author*: Your Name <@yourGitHubID> IRC: handle (if different)
```
The official spelling of GitHub is with a capital H.

Reply:
I have changed the spelling according to your suggestion @ felixfontein  , thanks

Reply:
Thanks! Could you please change the commit message though and remove the `@`? I don't want to be notified anytime this commit is used anywhere.

Reply:
> Could you please change the commit message though and remove the @? I don't want to be notified anytime this commit is used anywhere.

Jeez, I didn't know @felixfontein :-# sorry, forcepushed a commit message without the @

Reply:
No problem! I've also done this wrong in the past. In some of the repositories where we have a bot, the bot starts screaming once you do that to stop you from doing it, but here we don't have a bot ;)